server/handlers/k8sconfig_handler.go

```diff
@@ -265,6 +265,7 @@ func (h *Handler) LoadContextsAndPersist(token string, prov models.Provider) ([]
 		ctxName := "in-cluster"
 
 		cc, err := models.NewK8sContextFromInClusterConfig(ctxName, mid)
+		cc.DeploymentType = "in_cluster"
```

Comment:
@Revolyssup We need to consider a different name, WDYT?? // @leecalcote

Reply:
I've had trouble in the past thinking of a better descriptor for this deployment model. I've run out of ideas...